config.yaml.wmse

```diff
@@ -46,9 +46,10 @@ wiki:
               budget: budget
               finansiär_2: funder_2
               budget_2: budget_2
+          year_parameter: år
```

Comment:
This patch implements the year parameter differently from #20. The new implementation only passes year to _Projektdata_ but the other subpages also expect it. For the categorisation if nothing else.

Reply:
Missed that. Went back to the old implementation.